src/quo/components/tabs/tabs/schema.cljs

```diff
@@ -0,0 +1,32 @@
+(ns quo.components.tabs.tabs.schema)
+
+(def ^:private ?data-schema
+  [:sequential
+   [:maybe
+    [:map
+     [:id [:maybe :keyword]]
```

Comment:
About the `:maybe` here, wouldn't it be a bug to have a tab item without an ID? Seems really odd to rely on a nil ID, but maybe that's how it already is and the schema is just explicitly telling us now this oddity?